armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -153,6 +153,7 @@ def axiallyExpandAssembly(self, thermal: bool = False):
         mesh = [0.0]
         numOfBlocks = self.linked.a.countBlocksWithFlags()
         for ib, b in enumerate(self.linked.a):
+            runLog.debug(msg="Block {0}".format(b))
```

Comment:
I leave it to you to decide.  But do we want these `debug` statements here for all time?  

Or are these only useful to you during your dev process?

Reply:
They certainly originated from the dev process. I found them to be particularly useful for debugging and suspect that users would too if their results aren't producing any obvious errors and expected results. Happy to take them out/modify them if users don't like them but I think they have utility.

Reply:
Addressed https://github.com/terrapower/armi/pull/670/commits/138bef7694cfa3b360b67775e5fcec140f1515c1